CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Mapbox welcomes participation and contributions from everyone.
 ## Unreleased
 #### Features
 #### Bug fixes and improvements
+- Fixed an issue where `RoutingTilesOptions#tilesBaseUri` was used as a base url for route refresh requests instead of `RouteOptions#baseUrl`.
```

Comment:
Maybe mention the fact that we won't patch all the SDKs versions?
```suggestion
- Fixed an issue where `RoutingTilesOptions#tilesBaseUri` was used as a base url for route refresh requests instead of `RouteOptions#baseUrl`. This fix will be available only in 2.7.4 and after 2.15.0
```

Reply:
Let's add it as a known issue to all the other versions (in a follow-up PR, there' no future versions changelog here anyway). If I were a user and was bumping the SDK version, I would look at this version's changelog, not at some old one trying to find whether there were some fixes that are not present in the one I'm updating to.